sope.spec

```diff
@@ -14,7 +14,8 @@
 Summary:      SOPE
 Name:         sope%{sope_major_version}%{sope_minor_version}
 Version:      %{sope_major_version}.%{sope_minor_version}
-Release:      %{sope_release}%{?dist}
+Epoch:        1
```

Comment:
Here is the Epoch... 

NethServer/pull/11

Reply:
ok for me, approved, please merge it, and thinkt to approve my PR too